### DISCUSSION TITLE
avrc_metadata printing on Nextion Display

### DISCUSSION THREAD

**Discussion by Hellmas83** [pschatzmann/ESP32-A2DP#429] [Q&A] [answered]
Hello Thanks for this great Library it works realy good.

Iam new in coding. I want to get  avrc_metadata Titel and want to print it out on a Nextion Display. 

There is my little code

void avrc_metadata_callback(uint8_t typ, const uint8_t *text) {
  if (typ== ESP_AVRC_MD_ATTR_TITLE){
    
   myNex.writeStr("bttitel.txt",text);         
    
  }
}

But i become this error from my compiler

exit status 1

invalid conversion from 'const uint8_t* {aka const unsigned char*}' to 'const char*' [-fpermissive]

Can you help me?

**Comment by pschatzmann** [comment, marked as answer]
Just do a cast to convert to the requested type!

**Comment by Hellmas83** [comment]
it works thank you....
